MoneyDataType/LegacyAmountConverter.cs

```diff
@@ -1,107 +1,111 @@
-using System;
 using Money.Abstractions;
+using Newtonsoft.Json;
+using System;
+
+namespace Money.Serialization;
 
-namespace Money.Serialization
+internal class LegacyAmountConverter : JsonConverter<Amount>
 {
-    internal class LegacyAmountConverter : Newtonsoft.Json.JsonConverter<Amount>
-    {
-        private const string ValueName = "value";
-        private const string CurrencyName = "currency";
+    private const string ValueName = "value";
+    private const string CurrencyName = "currency";
+
+    private const string Name = "name";
+    private const string NativeName = "nativename";
+    private const string EnglishName = "englishname";
+    private const string Symbol = "symbol";
+    private const string Iso = "iso";
+    private const string Dec = "dec";
```

Comment:
See my remarks in [`AmountConverter`](https://github.com/OrchardCMS/OrchardCore.Commerce/pull/119/files#diff-b696f1e4c491c0eb40ad435484d95914095d6c6b857489883fcc80bfbad91cafR17).